crates/iota-core/Cargo.toml

```
@@ -22,7 +22,7 @@ bincode.workspace = true
 bytes.workspace = true
 count-min-sketch = "0.1.7"
 dashmap.workspace = true
-diffy = { version = "0.3", default-features = false }
+diffy = { workspace = true, default-features = false }
```

```suggestion
diffy.workspace = true
```
`default-features = false` in a member crate dependency has no effect and may be misleading. Default features are enabled/disabled in workspace manifest.